datasets/ONS-UK-trade-in-services/main.py

```diff
@@ -1,58 +1,196 @@
 #!/usr/bin/env python
 # coding: utf-8
 # %%
+from gssutils import *
+import json
+
+cubes = Cubes("info.json")
+scraper_1 = Scraper('https://www.ons.gov.uk/businessindustryandtrade/' + \
+                  'internationaltrade/datasets/uktradeinservicesallcountriesnonseasonallyadjusted')
+scraper_1
+trace = TransformTrace()
+
+# %%
+info = json.load(open("info.json"))
+scraper_1.dataset.family = info["families"]
+
+# %%
+dist_1 = scraper_1.distributions[0]
+dist_1
+
+# %%
+first_url = dist_1.downloadURL
+first_url
+
+# %%
+tabs = {tab.name: tab for tab in scraper_1.distribution(latest=True, mediaType=Excel).as_databaker()}
+
+
+# %%
+tab = tabs["TiS by country"]
```

Comment:
Remove this line. Duplicate tab selection.

Reply:
Removed duplicate tab selection